first.c

```diff
@@ -0,0 +1,112 @@
+
+#include <stdio.h>
+#include <ctype.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+
+void removeWhitespaces(char *exp)
+{
+    char *p = exp;
+    char *q = exp;
+    while (*q != '\0')
+    {
+        if (!isspace(*q))
+        {
+            *p++ = *q;
+        }
+        q++;
+    }
+    *p = '\0';
+};
+
+bool checkValidity(char *exp)

```

Comment:
method name should be like **bool isValidExpression(...)**

Reply:
I will update the method name to isValidExpression for better clarity.